components/shared/Icon.js

```diff
@@ -30,7 +30,7 @@ const Icon = props => {
       xmlns="http://www.w3.org/2000/svg"
       className={props.className}
       onClick={props.onClick}
-      viewBox={`0 0 ${props.height || '24'} ${props.width || '24'}`}
+      viewBox={`-5 -9 ${props.height || '24'} ${props.width || '24'}`}
```

Comment:
The first 2 numbers for `viewBox` are the x and y positions, so here you are adjusting the positioning specifically for the current arrow svg. This `Icon` component is used all over the place to render svgs. If you look at your site with these changes you will see items like the social icons are all now skewed. 

Cause this is used all over the solution to fix the arrows here is to figure out the correct path for the svg and then size needed for these buttons. Keeping the x and y as 0s here so that all svgs render at 0,0.

Reply:
Yep. I mentioned the potential problem with the viewBox in the Issue #35. I missed seeing the effects on the social icons, but I did try to find any side effects. In the Issue, I mentioned that we might need to pass in the x and y through props, if nothing else.

Reply:
I updated the svg for the arrow in svgs.js and reverted the x and y back to 0, 0.